src/tools/perf/perftest_params.c

```diff
@@ -70,7 +70,7 @@ static void usage(const struct perftest_context *ctx, const char *program)
     printf("     -s <size>      list of scatter-gather sizes for single message (%zu)\n",
                                 ctx->params.super.msg_size_list[0]);
     printf("                    for example: \"-s 16,48,8192,8192,14\"\n");
-    printf("     -m <send mem type>[,<recv mem type>]\n");
+    printf("     -m <send mem type[:device id]>,<recv mem type[:device id]>\n");
```

Comment:
`<recv mem type[:device id]>` is still optional? so probably need to wrap in `[`
Maybe add info explaining what is `device_id`